server/utils/responseUtils.js

```diff
@@ -36,4 +36,14 @@ exports.validationErrorResponse = (res, errors) => {
         errors: errors
     };
     res.status(HTTP_STATUS_CODES.BAD_REQUEST).json(response);
-};
\ No newline at end of file
+};
+
+exports.failResponse = (res, message, data, moreData) => {
+    let response = {
+        status: false,
+        message: message
+    };
+    if (data || !data) response.data = data;
+    if (moreData) response = { ...response, ...moreData };
+    res.status(HTTP_STATUS_CODES.OK).send(response);
```

Comment:
The status code could be 404, as requested document is not found in DB, This can be used.

Reply:
will update :+1: